pkgs/applications/misc/intune-portal/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, libuuid
+, xorg
+, curlMinimal
+, openssl
+, libsecret
+, webkitgtk
+, libsoup
+, gtk3
+, atk
+, pango
+, glib
+, sqlite
+, zlib
+, systemd
+, msalsdk-dbusclient
+, pam
+}:
+stdenv.mkDerivation rec {
+  pname = "intune-portal";
+  version = "1.2303.10";
+
+  src = fetchurl {
+    url = "https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/i/${pname}/${pname}_${version}_amd64.deb";
+    hash = "sha512-ikaZSzN/1VBMIKTHUytisGk4JxIwzLMCZRchTlKt1dOotJExLbYmNFwckxmYEKRHjZLDPUKOumcI8bwavJyadA==";
+  };
+
+  nativeBuildInputs = [ dpkg ];
+
+  unpackCmd = ''
+    mkdir -p root
+    dpkg-deb -x $curSrc root
+  '';
+
+
+  buildPhase =
+    let
+      libPath = {
+        intune = lib.makeLibraryPath [ stdenv.cc.cc.lib libuuid xorg.libX11 curlMinimal openssl libsecret webkitgtk libsoup gtk3 atk glib pango sqlite zlib systemd msalsdk-dbusclient ];
```

Comment:
Could you make the lists one dependency per line? It will make diffs nicer to read and `git blame` easier to use in the future.